akka-persistence-typed-tests/src/test/scala/akka/persistence/typed/state/scaladsl/NullEmptyStateSpec.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.typed.state.scaladsl
+
+/*
+ * Copyright (C) 2018-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+import akka.actor.testkit.typed.TestKitSettings
+import akka.actor.testkit.typed.scaladsl._
+import akka.actor.typed.ActorRef
+import akka.actor.typed.Behavior
+import akka.persistence.typed.PersistenceId
+import com.typesafe.config.Config
+import com.typesafe.config.ConfigFactory
+import org.scalatest.wordspec.AnyWordSpecLike
+
+object NullEmptyStateSpec {
+
+  private def conf: Config = ConfigFactory.parseString(s"""
+    akka.loglevel = INFO
+    akka.persistence.state.plugin = "akka.persistence.state.inmem"
+    akka.persistence.state.inmem {
+      class = "akka.persistence.state.inmem.InmemDurableStateStoreProvider"
+      recovery-timeout = 30s
+    }
+    """)
+}
+
+class NullEmptyStateSpec
+    extends ScalaTestWithActorTestKit(NullEmptyStateSpec.conf)
+    with AnyWordSpecLike
+    with LogCapturing {
+
+  implicit val testSettings: TestKitSettings = TestKitSettings(system)
+
+  def primitiveState(persistenceId: PersistenceId, probe: ActorRef[String]): Behavior[String] =
+    DurableStateBehavior[String, String](persistenceId, emptyState = null, commandHandler = (_, command) => {
+      if (command == "stop")
+        Effect.stop()
+      else
+        Effect.persist(command).thenReply(probe)(_ => command)
+    }).withDurableStateStorePluginId("akka.persistence.state.inmem")
```

Comment:
Custom plugin id not needed since the default is set in config? (here and in more places)

Reply:
@johanandren It was not working with the default for `DurableStateStore` and @patriknw also faced the same. He said he would look into it. Till then this workaround ..

Reply:
Alright, sounds good to go then if it was already discussed. 👍

Reply:
I have a fix to push soon.